core/ir/opnd_api.h

```diff
@@ -1124,8 +1124,166 @@ enum {
     /** Platform-independent way to refer to stack pointer. */
     DR_REG_XSP = DR_REG_SP,
 #    endif
+#elif defined(RISCV64)
+    DR_REG_INVALID, /**< Sentinel value indicating an invalid register. */
+    DR_REG_X0,
```

Comment:
We want Doxygen comments for each of these.

Reply:
Done